dotcom-rendering/src/components/marketing/hooks/useChoiceCards.ts

```diff
@@ -45,7 +45,8 @@ function transformChoiceCardsAmountsToProduct(
 const useChoiceCards = (
 	choiceCardAmounts: SelectedAmountsVariant | undefined,
 	countryCode: string | undefined,
-	content: BannerTextContent,
```

Comment:
Small refactor here because now both the banner and the new front container use this hook.